src/utils/RPCUtils.ts

```diff
@@ -0,0 +1,16 @@
+import { assert } from "chai";
```

Comment:
`chai` is a devDependency and it's only for test.

```suggestion
import assert from "assert";
```